action/action_deserializer.go

```diff
@@ -6,31 +6,15 @@
 
 package action
 
-import (
-	"github.com/iotexproject/iotex-proto/golang/iotextypes"
-)
+import "github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 // Deserializer de-serializes an action
```

Comment:
Pls add some explanatory comments for `action_deserializer` and `block_deserializer`